src/index.ts

```diff
@@ -241,7 +266,7 @@ class LiskMigrator extends Command {
 			cli.action.start('Starting lisk-core v4');
 			try {
 				const network = NETWORK_CONSTANT[nodeInfo.networkIdentifier].name as string;
```

Comment:
Perhaps this check can be done initially at the top if the `autoStartLiskCoreV4` is enabled, to avoid unnecessary computations.
```suggestion
				const networkConstant = NETWORK_CONSTANT[nodeInfo.networkIdentifier];
				if (!networkConstant) throw new Error(`Unknown network detected. No NETWORK_CONSTANT defined for networkID: ${nodeInfo.networkIdentifier}.`);
				const network = networkConstant.name as string;
```